src/sass/components/_direction.scss

```
@@ -0,0 +1,22 @@
+.direction{
+    .direction-nav{
+        .direction-list{
+            height: 80px;
+            margin-bottom: 0;
+            @include flexbox;
+            @include justify-content(flex-start);
+            .direction-item{
+                a, span{
+                    font-size: 1.4rem;
+                    color: $textColor;
+                    margin-right: 6px;
+                }
+                &.active{
+                    a,span{
+                        color: $textHoverColor;
+                    }
+                }
+            }
+        }
+    }
+}
```

```suggestion
.direction {
    &-list {
       height: 80px;
       margin-bottom: 0;
       @include flexbox;
       @include justify-content(flex-start);
    } 
    
    &-item {
       a, span {
          font-size: 1.4rem;
          color: $textColor;
          margin-right: 6px;
        }
        
       &.active {
            a,span {
               color: $textHoverColor;
            }
       }
    }
}
```
Tránh lồng nhau, nesting quá nhiều. Ngoài ra để ý thêm convention
Viết như này
`.direction {}`
Thay vì viết liền nhau
`.direction{}`